stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,433 @@
+import '../../dist/feed-list.css';
+import '../../dist/avatar.css';
+
+export default {
+    title: 'Components/Feed List',
+    parameters: {
+        description: 'The feed list item control displays unformatted text accompanied by an optional user image.',
+        tags: ['f3', 'a11y', 'theme']
+    }
+};
+
+export const standard = () => `
+    <div class="fd-feed-list" aria-label="Feed List Item default example">
+        <div class="fd-feed-list__body">
+            <div 
+                class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list__thumb" 
+                style="background-image: url(http://lorempixel.com/460/620/people/7/)" 
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe">    
+            </div>
+             
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.
+                </p>
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+standard.storyName = 'Standard';
+standard.parameters = {
+    docs: {
+        storyDescription: 'The standard feed list item'

```

Comment:
The standard feed list item displays a user's name and written text, an image and an additional attribute (in this example, it is a time stamp).